app/src/main/java/com/bignerdranch/android/criminalintent/CrimeListFragment.kt

```diff
@@ -101,10 +111,11 @@ class CrimeListFragment: Fragment() {
         }
     }
 
-    private fun updateUI() {
-        val crimes = crimeListViewModel.crimes
+    private fun updateUI(crimes: List<Crime>) {
         adapter = CrimeAdapter(crimes)
         crimeRecyclerView.adapter = adapter
     }
+}
+
+
```

Comment:
You have two empty lines at the end of file here. By default code style guidelines, there should be a single empty line at the end of file

Reply:
Unnecessary line deleted